docs/providers/aws/events/cloudfront.md

```diff
@@ -147,6 +147,7 @@ functions:
     events:
       - cloudFront:
           eventType: viewer-response
+          includeBody: true
```

Comment:
Let's also list it at: https://github.com/serverless/serverless/blob/157be6c9b9fc26ff08c59b8289d1079b7a5dbc82/docs/providers/aws/guide/serverless.yml.md